webpack/assets/javascripts/react_app/constants.js

```diff
@@ -1,5 +1,32 @@
+// API
 export const STATUS = {
   PENDING: 'PENDING',
   RESOLVED: 'RESOLVED',
   ERROR: 'ERROR',
 };
+
+// Documentation
+export const FOREMAN_VERSION = '1.23';
```

Comment:
you have this value in the api headers, It would be best to maintain all of these values it in one place (e.g. auto fetch it over api).